src/main/app/src/app/zaken/zaak-create/zaak-create.component.ts

```diff
@@ -366,7 +366,7 @@ export class ZaakCreateComponent implements OnInit, OnDestroy {
         this.klantenService
           .readPersoon(this.inboxProductaanvraag.initiatorID)
           .subscribe((initiator) => {
-            this.initiator = initiator;
+            this.initiator = initiator as Klant;
```

Comment:
this is needed as we are now using `RestPersoon` while `Klant` is still an self-defined class in the FE. when moving over to more generated types this cast will become unnecessary